django/db/backends/base/features.py

```diff
@@ -194,6 +194,12 @@ class BaseDatabaseFeatures:
     # Does the backend require literal defaults, rather than parameterized ones?
     requires_literal_defaults = False
 
+    # Does the backend support functions in defaults
+    supports_functions_in_defaults = True
+
+    # Does the backend support the DEFAULT keyword in insert queries
+    supports_default_keyword_in_insert = True
```

Comment:
I think we do need this. Looking at the definition of `pk_default_value` for sqlite (and others) we return `NULL`. As I understand it, this `NULL` isn't a problem for a primary key, because they're always not nullable and therefore the `NULL` is replaced by the db default. In general, however, a nullable field with a default will use `NULL` when provided, not the default. When databases that support `DEFAULT` we can use that syntax, otherwise we need to provide the db default expression ourselves (in `Field.pre_save`) or omit the field from the insert statement (in `Model._save_table`).